src/main/user-impl/java/com/mysql/cj/jdbc/NonRegisteringDriver.java

```diff
@@ -75,6 +75,11 @@
  */
 public class NonRegisteringDriver implements java.sql.Driver {
 
+    protected static boolean acceptAwsProtocolOnly = false;
+
+    private static final String MYSQL_PROTOCOL = "jdbc:mysql:";
+    private static final String AWS_PROTOCOL = MYSQL_PROTOCOL + "aws:";
```

Comment:
probably can remove these two and use SINGLE_CONNECTION_AWS.getScheme() instead of AWS_PROTOCOL